src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,290 @@
+import { FC, useState } from "react";
+import { LxdInstance } from "types/instance";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+  Select,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { createInstance, fetchInstances } from "api/instances";
+import { isClusteredServer } from "util/settings";
+import { useSettings } from "context/useSettings";
+import { useQuery, useQueryClient } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchStoragePools } from "api/storage-pools";
+import { fetchClusterMembers } from "api/cluster";
+import { useNavigate } from "react-router-dom";
+import { instanceNameValidation, truncateInstanceName } from "util/instances";
+import { fetchProjects } from "api/projects";
+import { LxdDiskDevice } from "types/device";
+import InstanceLink from "pages/instances/InstanceLink";
+import { useEventQueue } from "context/eventQueue";
+
+interface Props {
+  instance: LxdInstance;
+  close: () => void;
+}
+
+export interface LxdInstanceDuplicate {
+  instanceName: string;
+  targetProject: string;
+  targetClusterMember?: string;
+  targetStoragePool: string;
+  allowInconsistent: boolean;
+  instanceOnly: boolean;
+  stateful: boolean;
+}
+
+const DuplicateInstanceForm: FC<Props> = ({ instance, close }) => {
+  const toastNotify = useToastNotification();
+  const { data: settings } = useSettings();
+  const isClustered = isClusteredServer(settings);
+  const controllerState = useState<AbortController | null>(null);
+  const navigate = useNavigate();
+  const eventQueue = useEventQueue();
+  const queryClient = useQueryClient();
+
+  const { data: projects = [], isLoading: projectsLoading } = useQuery({
+    queryKey: [queryKeys.projects],
+    queryFn: fetchProjects,
+  });
+
+  const { data: storagePools = [], isLoading: storagePoolsLoading } = useQuery({
+    queryKey: [queryKeys.storage],
+    queryFn: () => fetchStoragePools(instance.project),
+  });
+
+  const { data: clusterMembers = [], isLoading: clusterMembersLoading } =
+    useQuery({
+      queryKey: [queryKeys.cluster],
+      queryFn: fetchClusterMembers,
+      enabled: isClustered,
+    });
+
+  const { data: instances = [] } = useQuery({
+    queryKey: [queryKeys.instances],
+    queryFn: () => fetchInstances(instance.project),
+  });
+
+  const notifySuccess = (instanceName: string, instanceProject: string) => {
+    const message = (
+      <>
+        Created Instance <strong>{instanceName}</strong>.
+      </>
+    );
+
+    const actions = [
+      {
+        label: `${instanceName}`,
+        onClick: () =>
+          navigate(`/ui/project/${instanceProject}/instance/${instanceName}`),
+      },
+    ];
```

Comment:
Let's align the messages structure
```suggestion
    const actions = [
      {
        label: "Configure",
        onClick: () =>
          navigate(`/ui/project/${instanceProject}/instance/${instanceName}/configuration`),
      },
    ];
```